Content.Goobstation.Client/Clothing/Components/SealableClothingVisualsComponent.cs

```diff
@@ -16,6 +18,6 @@ public sealed partial class SealableClothingVisualsComponent : Component
     [DataField]
     public string SpriteLayer = "sealed";
 
-    [DataField]
-    public Dictionary<string, List<PrototypeLayerData>> VisualLayers = new();
-}
\ No newline at end of file
+    [DataField("clothingVisuals")]
```

Comment:
```suggestion
    [DataField]
```
setting the name in quotes to the same as the variable name does nothing.